docs/DEV_SPACES.md

```diff
@@ -0,0 +1,134 @@
+Setting Up Azure Dev Spaces for Remote Monitoring Services
+==========================================================
+
+The following should be done for any of the Remote Monitoring web services where you'd like to use [Azure Dev Spaces][azds] with Visual Studio. Examples below use `asamanager` as the microservice, but similar steps would be used for other services.
+
+*Remember that [Azure Dev Spaces][azds] is in preview and subject to change.*
+
+## Prerequisites
+
+1. Create a [Kubernetes cluster (AKS) enabled for Azure Dev Spaces](https://docs.microsoft.com/en-us/azure/dev-spaces/get-started-netcore-visualstudio#create-a-kubernetes-cluster-enabled-for-azure-dev-spaces)
+
+
+1. Install the [Visual Studio Tools for Kubernetes][azds-vs] extension.
+
+1. Pull the code from the repo: [azure-iot-pcs-remote-monitoring-dotnet](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet)
+
+
+## Create your Dev Space
```

Comment:
For the steps below, it would be great if we can have screen shots. The first item doesn't need it.

Reply:
Agreed. However, I didn't add them since the docs team is just going to rewrite all of this anyway (with this as a first draft hopefully). Also, unfortunately, I am not able to get through all the steps myself as I have some sort of permission issue.